benchmarks/aggregate.py

```diff
@@ -277,6 +278,36 @@ def get_pr_titles(args):
   return [titles, data_labels]
 
 
+def speedup_header(title: str, backend_name: str, args):
+  if args.format == 'tab':
+    return f'Speedup\n{title}\nover\n{args.baseline.capitalize()}\n{backend_name}'
+  return f'Speedup({title}/{args.baseline.capitalize()} {backend_name})'
+
+
+def modelname_header(model: str, args):
+  if args.format == 'tab':
+    return f'ModelName\n{model}'
+  return f'ModelName({model})'
+
+
+def percentile_header(title: str, p: str, args):
+  if args.format == 'tab':
+    return f'{title}\n{p}'
+  return f'{title} {p}'
+
+
+def pr_text(headers, rows, args):
+  if args.format == 'csv':
+    if headers:
+      headers[0] = f'# {headers[0]}'
+    print(','.join(headers))
+    for row in rows:
+      print(','.join([str(f) if f is not None else '' for f in row]))
+  elif args.format == 'tab':
+    print(
+        tabulate(rows, headers=headers, tablefmt='fancy_grid', floatfmt='.2f'))
+
+
 def pr_latest(results_map: Dict[str, Any], args, timestamps: List[str]):
   titles, data_labels = get_pr_titles(args)
   speedups = [[] for _ in titles]
```

Comment:
Can we add short comment about the structure of speedups array? Indices semantics are unclear at a first glance.

Reply:
Done, thanks.